x-pack/plugin/esql/qa/server/single-node/src/javaRestTest/java/org/elasticsearch/xpack/esql/qa/single_node/HeapAttackIT.java

```diff
@@ -168,11 +168,24 @@ public void testSmallConcat() throws IOException {
     }
 
     @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/99826")
+    // TODO: drop testHugeConcatRejected() once this is fixed
     public void testHugeConcat() throws IOException {
         initSingleDocIndex();
         assertCircuitBreaks(() -> concat(10));
     }
```

Comment:
Can we just replace the body of this with `testHugeConcatRejected`.

Reply:
Replaced.